README.md

```diff
@@ -60,14 +60,29 @@ PUT /{realm}/identity-provider/instances/{alias}
   ...
 }
 ```
+If you need to manage many identity providers and had a default fallback identity server for non matching domains
+```json
+PUT /{realm}/identity-provider/instances/{alias}
+{
+  ...
+  "config": {
+    "home.idp.discovery.isfallback": "true",
+    ...
+  },
+  ...
+}
+```
 
 Note that domains need to be separated by two hashtags (`##`).
 
 You can also use the [Admin CLI (kcadm)](https://www.keycloak.org/docs/latest/server_admin/#identity-provider-operations):
 ```shell
 kcadm.sh update identity-provider/instances/{alias} -s 'config."home.idp.discovery.domains"="example.com##example.net"'
 ```
-
```

Comment:
This would allow to set the flag to `true` on multiple identity providers. How should we handle that case?